stdlib/2/user.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for user (Python 2)
+
+# Docs: https://docs.python.org/2/library/user.html
+# Source: https://hg.python.org/cpython/file/2.7/Lib/user.py
+from typing import BinaryIO, Text, Union
+
+
+home: str
+pythonrc: Union[bytes, Text]
+f: BinaryIO
```

Comment:
@JelleZijlstra How do we deal with variables that may or may not exist?

Secondly is it okay if I include few more stubs in the same PR?